EIPS/eip-725.md

```diff
@@ -1,55 +1,93 @@
----
```

Comment:
Please note that it is required for the preamble to start and end with `---` rather than ` ``` `.